devicetypes/smartthings/zwave-rgbw-bulb.src/zwave-rgbw-bulb.groovy

```diff
@@ -0,0 +1,353 @@
+/**
+ *  Copyright 2020 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *  Z-Wave RGBW Bulb, adapted from Aeon LED Bulb 6 Multi-Color
+ *
+ *  Author: SmartThings
+ *  Date: 2020-01-27
+ */
+
+metadata {
+	definition (name: "Z-Wave RGBW Bulb", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.light", mnmn: "SmartThings", vid: "generic-rgbw-color-bulb") {
```

Comment:
@greens Is there a UI definition for lights with no RGB support?

Reply:
I don't know.  Not sure who to ask?  Probably no.

This seems to provide some naming guidance: https://smartthings.atlassian.net/wiki/spaces/WORKS/pages/660606090/Fingerprint+writing+rules

Do the reviewers have any input on a proper name?  This naming was meant to parallel zigbee-rgbw-bulb.groovy, which has name "ZibBZee RGBW Bulb".

Reply:
This was resolved in an offline conversation.  There was a thought that the Illumin bulb may be non-RGB, but it does support color.

Decision here is to keep scope to only RGBW bulbs in this task.